src/pages/news/news-list/news-page-item.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from 'react';
```

Comment:
Please, rename `news-list` to `news-item` and all files inside this folder

Reply:
renamed